### ISSUE TITLE
Documentation #pylint

### ISSUE THREAD

**Issue by reshnashrestha** [ubsuny/23-Homework3G3#50]
How to run pylint in python script? https://github.com/orgs/ubsuny/teams/cp23-1

**Comment by s4il3sh** [comment]
@reshnashrestha It's easy, first download your code (you will have a file named: file_name.py) and run pylint file_name.py. It will show you your code's quality.

Btw, what does the link you provided mean?

**Comment by reshnashrestha** [comment]
@s4il3sh Yeah, I tried as I am using VS code for doing all work, I installed pylint on it, it is successfully installed but when I check for version it says not recognized. That's why I am worried.

**Comment by reshnashrestha** [comment]
Oh that link was actually supposed to be a mention.
